app/test_util.py

```diff
@@ -38,6 +37,12 @@ def server_url(self, value):
 
     def get(self, uri):
         return webdriver.Firefox.get(self, self.server_url + uri)
+        webdriver.Firefox.get(self, self.server_url + uri)
```

Comment:
How can this ever be executed if it's right after a return statement?

Reply:
Yeah, that return is supposed to go.  Didn't grab that line of the patch, will do.